bcda/web/api.go

```diff
@@ -33,37 +32,12 @@ import (
 
 var qc *que.Client
 
-/*
-  	swagger:route GET /api/v1/ExplanationOfBenefit/$export bulkData bulkEOBRequest
-
-	Start explanation of benefit data export
-
-	Initiates a job to collect explanation of benefit data from the Blue Button API for your ACO.
-
-	Produces:
-	- application/fhir+json
-
-	Security:
-		bearer_token:
-
-	Responses:
-		202: BulkRequestResponse
-		400: badRequestResponse
-		401: invalidCredentials
-		429: tooManyRequestsResponse
-		500: errorResponse
-*/
-
-func bulkEOBRequest(w http.ResponseWriter, r *http.Request) {
-	bulkRequest("ExplanationOfBenefit", w, r)
-}
-
 /*
 	swagger:route GET /api/v1/Patient/$export bulkData bulkPatientRequest
 
-	Start patient data export
+	Start patient, explanation of benefit, and coverage data export
 
-	Initiates a job to collect patient data from the Blue Button API for your ACO.
+	Initiates a job to collect patient, explanation of benefit, and coverage data from the Blue Button API for your ACO.
```

Comment:
Maybe:
`Initiates a job to collect data from the Blue Button API for your ACO.  Support resource types are Patient, Coverage, and ExplanationOfBenefit.`